lzy/execution-env/src/main/java/ai/lzy/env/base/DockerEnvDescription.java

```diff
@@ -18,7 +18,9 @@ public record DockerEnvDescription(
     String networkMode,
     DockerClientConfig dockerClientConfig,
     String user,
-    Set<String> allowedPlatforms // In format os/arch like "linux/amd64". Empty means all are allowed
+    Set<String> allowedPlatforms, // In format os/arch like "linux/amd64". Empty means all are allowed
+    @Nullable
+    Long memLimitMb
```

Comment:
These can be put on one line

Reply:
no, no :)
we place annotations on separate lines